.github/workflows/ci.yml

```diff
@@ -34,19 +34,34 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2
 
+      - name: Start MongoDB
```

Comment:
I'm wondering if the tests which require running mongo should be moved out of regular unit tests. Since so far our unit tests have no requirements for external running software.